src/tests/system/tests/test_passkey.py

```diff
@@ -108,3 +108,140 @@ def test_passkey__su(client: Client, provider: GenericProvider, moduledatadir: s
         ioctl=f"{moduledatadir}/umockdev.ioctl",
         script=f"{testdatadir}/umockdev.script.{suffix}",
     )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.topology(KnownTopology.Samba)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__su_fail_pin(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
+    """
+    :title: Check authentication deny of user with LDAP, IPA, AD and Samba with incorrect pin
```

Comment:
Specify `su` in the description since that is the only mode of authentication tested here.   Could be "Check su authentication denies user with incorrect pin for LDAP, IPA, AD, and Samba"  or similar.